### PR TITLE
postgresql: build with ICU

### PR DESCRIPTION
**Pull request by ghost**
Refs #30097

- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by fxcoudert — CHANGES_REQUESTED** [review]
`revision 1` line needed after `sha256` line

File: Formula/postgresql.rb [hunk 1]
```diff
@@ -22,9 +22,11 @@ class Postgresql < Formula
   deprecated_option "enable-dtrace" => "with-dtrace"
   deprecated_option "with-python3" => "with-python"
 
+  depends_on "icu4c"
   depends_on "openssl"
   depends_on "readline"
 
+  depends_on "pkg-config" => :build
```
[the remark this answers is not in the document]
Comment (dtrebbien) [reply Formula/postgresql.rb]: Done

**Review by fxcoudert — APPROVED** [review]

**Comment by javian** [comment]
The error output is this
```
03:10:07 Error: libpqxx: failed
03:10:07 Please revision bump libpqxx.
```
Can you add a revision bump to the libpqxx formula ?

**Comment by dtrebbien** [comment]
Pushed a commit that bumps the revision of libpqxx

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/04974e62a451584703c34cbbcfdf0fcc4c9292d8 & https://github.com/Homebrew/homebrew-core/commit/cff87fc1dc053b80fd0d7a08ac18581f5f44d9ed. Thank you for your contribution to Homebrew @dtrebbien; we appreciate it! 😺

**Pull request closed without merging**